mlflow/entities/span_status.py

```diff
@@ -0,0 +1,53 @@
+from __future__ import annotations
+
+from dataclasses import dataclass
+
+from opentelemetry import trace as trace_api
+
+from mlflow.exceptions import MlflowException
+from mlflow.protos.databricks_pb2 import INVALID_PARAMETER_VALUE
+from mlflow.tracing.types.constant import TraceStatusCode
+
+
+@dataclass
+class SpanStatus:
+    """
+    Status of the span or the trace.
+
+    Args:
+        status_code: The status code of the span or the trace.
+        description: Description of the status. Optional.
+    """
+
+    status_code: TraceStatusCode
+    description: str = ""
+
+    @staticmethod
+    def to_otel_status(status: SpanStatus) -> trace_api.Status:
```

Comment:
Can this be an instance method?

Reply:
ah definitely, these were different static methods originally and I forgot to convert them:)